docs/0.1/creating_schemas.md

```diff
@@ -0,0 +1,283 @@
+This file sets up the nodes `alpha-node-000` and `beta-node-000` and runs the
+Grid and Splinter components in separate containers; these names appear in
+example prompts and commands. (See [Running Hyperledger Grid on
+Splinter]({% link docs/0.1/grid_on_splinter.md %}) for more information.)
+
+If you are not using this example environment, replace these items with the
+actual values for your environment when entering each command.
+
+### Connect to a Grid Node
+
+1. Connect to the first node's `gridd` container and start a bash session.
+
+   For the example environment, use this command to connect to the `gridd-alpha`
+   container and run Grid commands on `alpha-node-000`.
+
+   ```
+   $ docker exec -it gridd-alpha bash
+   root@gridd-alpha:/#
+   ```
+
+### Set Up Your Environment
+
+Set the following Grid environment variables to specify information for the
+`grid` commands in this procedure.
+
+{:start="2"}
+
+2. Set `GRID_DAEMON_KEY` to the base name of your public/private key files
+   (the example environment uses `alpha-agent`).  This environment variable
+   replaces the `-k` option on the `grid` command line.
+
+   ```
+   root@gridd-alpha:/# export GRID_DAEMON_KEY="alpha-agent"
+   ```
+
+   **Note**: Although this variable has "daemon" in the name, it should
+   reference the ***user key files*** in `$HOME/.grid/keys`, not the Grid
+   daemon's key files in `/etc/grid/keys`.
+
+1. Set `GRID_DAEMON_ENDPOINT` to the Grid daemon's endpoint (URL and port),
+   such as `http://localhost:8080`. This environment variable replaces the
+   `-url` option on the `grid` command line.
+
+   ```
+   root@gridd-alpha:/# export GRID_DAEMON_ENDPOINT="http://localhost:8080"
+   ```
+
+1. For Grid on Splinter, set `GRID_SERVICE_ID` to the fully qualified service
+   ID (such as `01234-ABCDE::gsAA`) for the scabbard service on the circuit.
+   This environment variable replaces the `--service_id` option on the `grid`
+   command line.
+
+   ```
+   root@gridd-alpha:/# export GRID_SERVICE_ID="01234-ABCDE::gsAA"
+   ```
+
+   Tip: See [Prerequisites](#prerequisites) for the `splinter` commands that
+   display the circuit ID and four-character service ID string.
+
+### Define and Create a Schema
+
+{:start="5"}
+
+5. Create a schema definition file, in YAML format, that specifies the
+   following information:
+
+   * Schema name, such as `gs1_product` (this string tells Grid that the schema
+     will use the GS1 namespace and will be associated with all GS1 products)
+   * Description
+   * Set of item properties (each with a name, data type, and value
+     that conforms to the data type)
+     <br><br>
+
+   The following example shows how to use `cat` to create the file
+   `product_schema.yaml`, with sample contents that match the product definition
+   shown in [Creating Products]({% link docs/0.1/creating_products.md %}).
+
+   ```
+   root@gridd-alpha:/# cat > product_schema.yaml
```

Comment:
Using `cat` to create these YAML files has caused issues in the past with incorrect formatting. I would suggest rewriting these steps to use `docker cp` similar to the grid on splinter example in the Grid repository.

Reply:
Thanks!  That's good to know.  For time reasons, this will need to be done in a separate story.